flowredux/src/commonMain/kotlin/com/freeletics/flowredux/sideeffects/OnEnterStateMachine.kt

```diff
@@ -20,7 +20,7 @@ import kotlinx.coroutines.flow.onStart
 import kotlinx.coroutines.launch
 
 @ExperimentalCoroutinesApi
-internal class StartStatemachineOnEnterSideEffectBuilder<SubStateMachineState : Any, SubStateMachineAction : Any, InputState : S, S : Any, A>(
+internal class OnEnterStateMachine<SubStateMachineState : Any, SubStateMachineAction : Any, InputState : S, S : Any, A>(
```

Comment:
Just personal preference: `OnEnterStartStateMachine` (so add the `Start`). Your call, just a personal preference.

Reply:
Makes sense [d2c9d88](https://github.com/freeletics/FlowRedux/pull/505/commits/d2c9d88aeafd019b88f8859559d1b73a052c849a)